### ISSUE TITLE
CI: Codecov coverage.xml upload fails

### ISSUE THREAD

**Issue by kentrutan** [pymoca/pymoca#332]
We are not getting the email report on coverage changes from Codecov and can't dig into the detail on codecov.io because the coverage.xml upload to their server is failing. The same error happens after retries. This is the context from the GitHub action output:
```
[2024-06-21T17:58:10.792Z] ['info'] Processing ./coverage.xml...
[2024-06-21T17:58:10.798Z] ['info'] Detected GitHub Actions as the CI provider.
[2024-06-21T17:58:11.283Z] ['info'] Pinging Codecov: https://codecov.io/upload/v4?package=github-action-3.1.6-uploader-0.7.3&token=*******&branch=fix-inherited-symbol-scope-pr&build=9617740500&build_url=https%3A%2F%2Fgithub.com%2Fpymoca%2Fpymoca%2Factions%2Fruns%2F9617740500%2Fjob%2F26530283915&commit=c2a897876e0eda1be5889ea190397fd78917998a&job=CI&pr=307&service=github-actions&slug=pymoca%2Fpymoca&name=&tag=&flags=&parent=
[2024-06-21T17:58:11.511Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 429 - {'detail': ErrorDetail(string='Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 2461s.', code='throttled')}
```

**Comment by jackvreeken** [comment]
This was already addressed by #330, but I think I forgot to ask for you (quick) review there :)

**Comment by kentrutan** [comment]
Fixed by #330.
